cypress/apiRequests/storeFront/Collections.js

```diff
@@ -0,0 +1,21 @@
+class Collections {
+  getCollection(collectionId, channelSlug) {
+    const query = `query Collection{
+        collection(id: "${collectionId}", channel: "${channelSlug}") {
+          id
+          slug
+          name
+          products(first:100){
+            edges{
+              node{
+                id
+                name
+              }
+            }
+          }
+        }
+      }`;
+    return cy.sendRequestWithQuery(query, "token");
```

Comment:
Is token needed here?

Reply:
Yes. I want to check this collection like guest user on the store. Without this token it will send this request with dashboard auth. With token request will not return hidden collections, and without it, request will return hidden collections.